xml_converter/proto/waypoint.proto

```diff
@@ -297,7 +297,7 @@ message SpeciesFilter {
 }
 
 message TrailData {
-    string trail_data = 1;
+    string trail_data_relative_path = 1;
```

Comment:
As discussed. This value doesn't need to be saved because xml_convertor will write the trail files when exporting XML and can be saved in a known standardized location